### ISSUE TITLE
Need for remove file option

### ISSUE THREAD

**Issue by vbarrera** [snowformatics/BlasterQt#12]
Hi.

When using the tool, if you provide a file for any of the section, it can't be removed. If used afterwards using the text field, it continues using the file rather than the text file so the results don't change.

Best.

**Comment by snowformatics** [comment]
I either don't understand what you mean or I can not reproduce the problem with my test sequences. The text edit field and the file button are connected. If you load a file the content will be inserted into the text field.

I added two screenshots, the first one shows a Bowtie run with a file and for the second I removed the last 3 nucleotide of the second sequence inside the text field. Both output show the expected behavior.
![02](https://user-images.githubusercontent.com/5192670/63796004-7b2bd380-c905-11e9-9d48-cf9385bcf14b.png)
![01](https://user-images.githubusercontent.com/5192670/63796005-7bc46a00-c905-11e9-905c-fdba1c811102.png)
